### PR TITLE
Add jobrunner setup and management scripts

### PR DESCRIPTION
**Pull request by bloodearnest**
- creates shared jobrunner user, in docker group
- code goes in /srv/jobrunner/code
- dependencies go in /srv/jobrunner/lib
- config goes in various files in /srv/jobrunner/environ
- output directories in /srv/{medium,high}_privacy
- careful ownership and permissions on above
- systemd service

### CODE REVIEW

File: jobrunner/secrets-template.env [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# These are secrets that should not be kept privately
```
Comment (sebbacon) [inline jobrunner/secrets-template.env:1]: This is a comment which does not quite make sense ^

Should they be kept publicly? :)

File: jobrunner/secrets-template.env [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+# Credentials for logging into the job server
+JOB_SERVER_TOKEN=
+
+# A Github developer token that has read access to private repos
+PRIVATE_REPO_ACCESS_TOKEN=
+
+# A database containing dummy data; results from this could be freely
+# published without review
+DUMMY_DATABASE_URL=
```
Comment (sebbacon) [inline jobrunner/secrets-template.env:11]: We've basically decided to get rid of this. We're no longer running the dummy TPP SQL Server, I *think* it's no longer an option in the job server, and we're going to support custom expectations-based dummy backends at Some Point

File: jobrunner/secrets-template.env [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+# A database containing a slice of the full data; useful for checking
+# or debuggin real data without potentially having to wait for hours
+# for completion
+SLICE_DATABASE_URL=
+
+# The full database
+FULL_DATABASE_URL=
+
+# License for running stata images
```
Comment (sebbacon) [inline jobrunner/secrets-template.env:21]: Is this a path to a license or a license key?

File: scripts/jobrunner.sh [hunk 4]
```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# Set up the job-runner service
+set -euo pipefail
+BACKEND=$1
+
+# set default file creation permission for this script be 640 for files and 750
+# for directories
+umask 027
+
+# ensure shared sysytem user is set up properly
```
Comment (sebbacon) [inline scripts/jobrunner.sh:10]: tpyo

File: tests/emis.sh [hunk 5]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-./emis/manage.sh
+./emis-backend/manage.sh
 #run again to check for idempotency
-./emis/manage.sh
+./emis-backend/manage.sh
```
Comment (sebbacon) [inline tests/emis.sh:4]: Add a trivial assertion here like you did with TPP?

**Review by sebbacon — APPROVED** [review]
This is very exciting!

File: jobrunner/secrets-template.env (hunk 1 above)
Comment (bloodearnest) [reply jobrunner/secrets-template.env:1]: "should be kept private" was what I think I was going for

File: jobrunner/secrets-template.env (hunk 2 above)
Comment (bloodearnest) [reply jobrunner/secrets-template.env:11]: Ok, will drop it.

I have a vague plan for better config documentation and validation, but it's not top priority

File: jobrunner/secrets-template.env (hunk 3 above)
Comment (bloodearnest) [reply jobrunner/secrets-template.env:21]: The key itself as a string - will document

File: tests/emis.sh (hunk 5 above)
Comment (bloodearnest) [reply tests/emis.sh:4]: will do

**Pull request merged**